ui/js/models/resultset.js

```diff
@@ -1,332 +0,0 @@
-                            return response.data.results;
-                        }
-                        return $q.reject('No results found for revision ' +
-                            revision + ' on project ' +
-                            projectName);
-                    });
-            },
-
-            cancelAll: function (resultset_id) {
-                const uri = resultset_id + '/cancel_all/';
-                return $http.post(getProjectUrl('/resultset/') + uri);
-            },
-
-            triggerMissingJobs: function (decisionTaskId) {
-                return taskclusterModel.load(decisionTaskId).then((results) => {
-                    const actionTaskId = slugid();
-
-                    // In this case we have actions.json tasks
-                    if (results) {
-                        const missingtask = results.actions.find(action =>
-                            action.name === 'run-missing-tests');
-                        // We'll fall back to actions.yaml if this isn't true
-                        if (missingtask) {
-                            return taskclusterModel.submit({
-                                action: missingtask,
-                                actionTaskId,
-                                decisionTaskId,
-                                taskId: null,
-                                task: null,
-                                input: {},
-                                staticActionVariables: results.staticActionVariables,
-                            }).then(() => `Request sent to trigger missing jobs via actions.json (${actionTaskId})`);
-                        }
-                    }
-                });
-            },
-
-            triggerAllTalosJobs: function (times, decisionTaskId) {
-                return taskclusterModel.load(decisionTaskId).then((results) => {
-                    const actionTaskId = slugid();
-
-                    // In this case we have actions.json tasks
-                    if (results) {
-                        const talostask = results.actions.find(action =>
-                            action.name === 'run-all-talos');
-                        // We'll fall back to actions.yaml if this isn't true
-                        if (talostask) {
-                            return taskclusterModel.submit({
-                                action: talostask,
-                                actionTaskId,
-                                decisionTaskId,
-                                taskId: null,
-                                task: null,
-                                input: { times },
-                                staticActionVariables: results.staticActionVariables,
-                            }).then(function () {
-                                return `Request sent to trigger all talos jobs ${times} time(s) via actions.json (${actionTaskId})`;
-                            });
-                        }
-                    }
-
```

Comment:
Talked to @imbstack and @jmaher : This block of code wasn't migrated over.  It only applies to ``Esr52`` and ``Talos`` jobs are no longer run against that branch.  It expires in Sept, anyway.